project-koku/masu
# Switching OCP data from GB to GiB

Updating summary calculation to use GiB instead of GB (1GiB ≈ 1.074GB)

*Testing:*
1. Ingest OCP data with master and run Koku API query
2. Switch masu to branch with GiB change and refresh summary data
3. Run same query from 1 and verify that the numbers were updated by the approximation of 1GiB ≈ 1.074GB

*Test Result:*
[masu_gib_convert_ut.txt](https://github.com/project-koku/masu/files/2692445/masu_gib_convert_ut.txt)

Closes #278
